custom_components/tahoma/__init__.py

```diff
@@ -87,8 +89,18 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
 
+    if entry.data.get(CONF_HUB):
+        endpoint = SUPPORTED_ENDPOINTS[entry.data.get(CONF_HUB)]
+    else:
+        endpoint = None
```

Comment:
Can we fallback to the classic endpoint instead? It will ease to understand the code.

Reply:
@tetienne good one, I will simplify this. When passing 'None', it will default to the tahomalink.com endpoint since it is default behaviour of pyhoma. However this is a bit like a black box in this way indeed.

Reply:
@tetienne I believe I solved in a better way now.